### DISCUSSION TITLE
Eksplorasi web scraping lowongan kerja (jobdesc, pindah halaman, detail pekerjaan tiap lowongan)

### DISCUSSION THREAD

**Discussion by cakmustafagit** [teamalgoritma/community#761] [Q&A] [answered]
Berdasarkan penugasan dalam capstone proyek web scraping, kami diminta untuk scraping data pekerjaan yang ada di link : https://www.kalibrr.id/job-board/te/data/1 sampai dengan halaman 15. Namun untuk eksplorasi lebih lanjut diluar penugasan capstone proyek, ada beberapa hal yang ingin saya tanyakan :
1. Berdasarkan informasi dari link diatas, bagaimana caranya ambil data jobdesc yang dalam class div nya tidak muncul, capture terlampir.
![image](https://github.com/teamalgoritma/community/assets/94230963/103a3b7e-4fed-48a8-ad5c-84cf991d393f)

2. Cara untuk scraping halaman berikutnya, sudah mencoba-coba namun belum berhasil.
3. Cara untuk scraping data berdasarkan detail dari tiap lowongan kerja setelah di klik. Capture terlampir
  
a. klik detail informasi lowongan
![image](https://github.com/teamalgoritma/community/assets/94230963/b538d41d-7fd2-4cda-9871-af8c8e9398f3)

b. Ambil informasi di detail
- deskripsi pekerjaan
![image](https://github.com/teamalgoritma/community/assets/94230963/d20f509f-04e0-437f-ba90-a10c3ee13b81)

- kualifikasi, keahlian,  dan informasi lain yang ada di detail tersebut
![image](https://github.com/teamalgoritma/community/assets/94230963/381d488c-1f4a-4b85-984c-ddd94e6bc4e5)

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @cakmustafagit,

Berikut adalah cara-cara yang dapat dilakukan untuk menjawab pertanyaan bapak:
1. Caranya sebenarnya sama dengan mengambil data yang lainnya, namun untuk attrs nya dapat diisi `"class"=""`
2. Untuk mengambil link halaman selanjutnya, dapat menggunakan for loop sebanyak 15x pada seluruh proses dari pemilihan link, namun angka di belakang link nya diubah sesuai dengan nilai `i` pada tiap loop, `"https://www.kalibrr.id/job-board/te/data/" + i`
3. Ketika di klik, maka link website yang perlu di scrap berubah, berbeda dengan link yang isinya list pekerjaan di halaman utama. Sehingga, data tersebut tidak bisa diambil secara sekaligus.

Semoga jawaban saya membantu!

**Reply by cakmustafagit** [reply]
Terimakasih..., saya coba dulu
